packages/components/src/components/post-mainnavigation/post-mainnavigation.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, Event, EventEmitter, Host, Listen, Method, h } from '@stencil/core';
+
+@Component({
+  tag: 'post-mainnavigation',
+  shadow: false,
+  styleUrl: './post-mainnavigation.scss',
+})
+export class PostMainnavigation {
+  /**
+   * Gets emitted when a user closes the main navigation on mobile
+   */
+  @Event() postMainNavigationClosed: EventEmitter;
```

Comment:
To align with other events: 
```suggestion
  @Event() postToggle: EventEmitter;
```